### PR TITLE
Use ProjectService in getScalingRiskEntry

### PR DESCRIPTION
**Pull request by sz-piotr**

### CODE REVIEW

File: packages/config/src/projects/project/Project.ts [hunk 1]
```diff
@@ -11,18 +13,60 @@ export interface Project {
...
+  daLayer: string
+  stage: string
+  purposes: string[]
+}
+
+export interface ProjectScalingRisks {
+  self: ScalingProjectRiskView
+  host: ScalingProjectRiskView | undefined
+  stacked: ScalingProjectRiskView | undefined
+}
```
Comment (torztomasz) [inline packages/config/src/projects/project/Project.ts:64]: maybe better to use union?
```suggestion
export type ProjectScalingRisks = {
  type: 'single'
  self: ScalingProjectRiskView
} | {
  type: 'multiple'
  host: ScalingProjectRiskView
  stacked: ScalingProjectRiskView
}
```

File: packages/frontend/src/server/features/scaling/get-common-scaling-entry.ts [hunk 2]
```diff
@@ -102,11 +166,11 @@ export function getCommonScalingEntry({
   }
 }
 
-function getStageOrder(stage: StageConfig | undefined): number {
-  if (stage?.stage === 'Stage 2' || stage?.stage === 'Stage 1') {
+function getStageOrder(stage: string | undefined): number {
```
Comment (torztomasz) [inline packages/frontend/src/server/features/scaling/get-common-scaling-entry.ts:169]: can't we have it as StageConfig? I would prefer to have the checks below typesafe

**Review by torztomasz — APPROVED** [review]

File: packages/config/src/projects/project/Project.ts (hunk 1 above)
Comment (sz-piotr) [reply packages/config/src/projects/project/Project.ts:64]: I errored on the ease of use. Also when I was writing there was this edge case of a project having stacked risks but no host risks. After removing hostChain = multiple this is no longer the case so maybe union? But then it's harder to use.

File: packages/frontend/src/server/features/scaling/get-common-scaling-entry.ts (hunk 2 above)
Comment (sz-piotr) [reply packages/frontend/src/server/features/scaling/get-common-scaling-entry.ts:169]: I'm going to make another PR where I make some of those small things cleaner

**Pull request merged**